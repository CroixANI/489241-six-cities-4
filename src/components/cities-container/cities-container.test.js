import React from 'react';
import renderer from 'react-test-renderer';

import CitiesContainer from './cities-container';

const CITIES = [`Minsk`, `Brest`, `Grodno`];

it(`Render Map component`, () => {
  const tree = renderer
    .create(<CitiesContainer activeItem={CITIES[0]} items={CITIES} onItemSelected={() => {}} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
