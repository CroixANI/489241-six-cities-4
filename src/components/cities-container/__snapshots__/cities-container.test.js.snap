// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Map component 1`] = `
<div
  className="tabs"
>
  <section
    className="locations container"
  >
    <ul
      className="locations__list tabs__list"
    >
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="#"
        >
          <span>
            Brest
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="#"
        >
          <span>
            Grodno
          </span>
        </a>
      </li>
      <li
        className="locations__item"
      >
        <a
          className="locations__item-link tabs__item"
          href="#"
        >
          <span>
            Minsk
          </span>
        </a>
      </li>
    </ul>
  </section>
</div>
`;
